apps/explorer/src/hooks/useFormatCoin.ts

```diff
@@ -66,7 +66,7 @@ export function useCoinDecimals(coinType?: string | null) {
                 );
             }
 
-            return rpc.getCoinDenominationInfo(coinType);
+            return rpc.getCoinMetadata(coinType);
```

Comment:
@666lcz what happens if this method isn’t available? This is currently hardcoded for SUI so I’m worried that this will break sui display for networks that don’t yet support this

Reply:
Good catch! I just added the logic to return hardcoded value for SUI if the RPC version is prior to `0.17.0`